### PR TITLE
Add (core\db): Support for Homebind Orientation

### PR DESCRIPTION
**Pull request by acidmanifesto**
## Changes Proposed:
-  Adds Orientation Collumn for Character_Hombind in the Character DB
-  Supports Orientation save and load for  hearthstone use and homebind use
- Exisiting Orientation is set to zero for homebinds without a orientation set so exsisting homebinds are not messed with.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/13386
- Closes https://github.com/chromiecraft/chromiecraft/issues/4094
## Tests Performed:

- builds
- self tested but don't take my word for it

## How to Test the Changes:

1. do a .cheat cooldown so you wont rage with the cool down
2. homebind while facing some weird direction so you dont think its a fluke
3. go somewhere and face a different way then what you were when you binded your hearthstone.
4. use hearthstone
5. observe orientation was as it was when you set the homehind

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by acidmanifesto** [comment]
paging @heyitsbench  for test.

File: data/sql/updates/db_characters/Homebind_Orientation.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+--
+
+ALTER TABLE `character_homebind`
+	ADD COLUMN `posO` FLOAT NOT NULL DEFAULT '0' AFTER `posZ`;
```
[the remark this answers is not in the document]
Comment (acidmanifesto) [reply data/sql/updates/db_characters/Homebind_Orientation.sql]: i did a rebase just as you posted. lol.

**Comment by heyitsbench** [comment]
Tested, works as described. 👍

**Comment by acidmanifesto** [comment]
> Tested, works as described. 👍

Thanks for the test.

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**